gemini/use-cases/retrieval-augmented-generation/rag_vector_embedding_in_bigquery.ipynb

```diff
@@ -0,0 +1,794 @@
+{
```

Comment:
<p>mentioned in <a href="https://cloud.google.com/bigquery/docs/vector-index-text-search-tutorial" target="_blank">Perform </a> ...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/487/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Changed</p>